### ISSUE TITLE
Only String argument types is working when sending transaction through through the wallet

### ISSUE THREAD

**Issue by psiemens** [flow-hydraulics/flow-wallet-api#153]
This issue was originally opened here by @m4nuC on August 20, 2021: https://github.com/onflow/wallet-api/issues/15

> When sending a transaction it seems that only the string type is currently working.
>
> ie.: this will work
```
{
  "code":"transaction(greeting: String) { execute { log(greeting.concat(\", World!\")) }}",
  "arguments":[{"type":"String","value":"Hello"}]
}
```

> But this won't
```
{
  "code":"transaction(greeting: UInt64) { execute { log(greeting.concat(\", World!\")) }}",
  "arguments":[{"type":"UInt64","value":"Hello"}]
}

```
> Error is: 
> 
> `Error while processing job: failed to decode value: invalid JSON Cadence structure`

**Comment by nanuuki** [comment]
The provided example has the value `"Hello"` and `UInt64` type, which is invalid - hence the error message `"invalid JSON Cadence structure"`.

Here is an example of using an `UInt64` argument (just log the value):

```
{
  "code":"transaction(greeting: UInt64) { execute { log(greeting) }}",
  "arguments":[{"type":"UInt64","value":"1"}]
}
```

@m4nuC, can you confirm if it works for you?

**Comment by nanuuki** [comment]
This has been resolved in #196
